search/LowestCommonAncesstor2RMQ/rmq_pm1.cpp

```diff
@@ -0,0 +1,205 @@
+#include "rmq_pm1.hpp"
+
+using baron::RMQPM1;
```

Comment:
Not sure if such nesting is needed

Reply:
Okay, I've also namespaced the RMQ class with "baron". I presume we won't use a namespace for the library?